docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/specialReport/SpecialReport.scss

```diff
@@ -22,8 +22,21 @@
     overflow: hidden;
     height: 59.375vw;
 
+
+    @include bp(xtra-small) {
+      max-height: 384px;
+      height: 384px;
+    }
+
+    @include break(small) {
```

Comment:
why are you using a combination of bp() and break() mixins? Let's stick with mobile-first break()